
extern void error_catch(void);
